### DISCUSSION TITLE
Lens and No Lens Comparison

### DISCUSSION THREAD

**Discussion by OpenScanFan** [OpenScanEu/OpenScan2#26] [Show and tell]
Just thought I'de share this

I have been experimenting with different lens types, this is the setup that I used:

Scanner: OpenScan Mini
Camera: IMX519
Ringlight Used: No
Rotor Angles: min -40 Max 60
#of images used: 600 (2 sets of 300) focus stacked
Additional Lenses Used: x2 40.5mm Close Up Lenses with a +2 and +4 Diopter
*Note: Please bear in mind that this is just a TEST result, I have not finalized anything on a correct lens option yet

There are a couple problems with using lenses though:

1. The camera itself comes with a fitted standard lens, of which the specs are unknown to me and is not removeable
2. The camera does not come with any kind of lens mount option
3. Fitting lenses to the front of the camera is a botch job
4. There seems to be a problem with the DOF (Depth of Field) additional lenses seems to limit the focal plane to a degree that the camera cannot focus to infinity

x4 25mm Brass Standoff's to bring the camera closer to the subject

Here is a comparison of a small part with and without the lenses
![Part_Lens_Comparison](https://user-images.githubusercontent.com/101347768/158975587-a7ac6115-cb68-4f87-83ec-c46fe3f4b737.jpg)

**Comment by sokol07** [comment]
What do you mean by with and without lens? I mean, what lens? And what camera was that scanned with?
How did you do the stacking? Did you feed all the images or did you stack them in 3rd party software?
Sorry for so many questions, but the difference is amazing, the details rendition on the right scan is great.

**Reply by OpenScanFan** [reply]
All good mate, questions are good, that's how we get to know things ;-)

Thank you for your kind words, made me happy :-)

I just updated the info, you can check it out
I used the IMX519, focus bracketing is a feature in the scan software, downloaded images, and batch processed (stacked) them in an external software

There are a couple problems with using lenses though:
1. The camera itself comes with a fitted standard lens, of which the specs are unknown to me and is not removeable
2. The camera does not come with any kind of lens mount option
3. fitting lenses to the front of the camera is a botch job

**Reply by sokol07** [reply]
Do I understand correctly: you placed the additional lenses in front of the standard IMX519 lens?
Nice idea, I have to try this...

**Reply by OpenScanFan** [reply]
yes, that is correct
